src/main/java/duke/command/CommandType.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+/**
+ * The enum Command type.
+ */
+public enum CommandType {
+    /**
+     * Bye command type.
+     */
+    BYE,
+    /**
+     * List command type.
+     */
+    LIST,
+    /**
+     * Delete command type.
+     */
+    DELETE,
+    /**
+     * Done command type.
+     */
+    DONE,
+    /**
+     * Task command type.
+     */
+    TASK,
+    /**
+     * Find command type.
+     */
+    FIND
+}
```

Comment:
Might not be necessary to indicate all the command types in javadocs.

Reply:
Hmm I think all the comments in relation to javadocs is bcs I autogenerated javadocs for all elements and just edited it afterwards. I personally think that this looks nicer when I open the javadocs .html pages, maybe not in the .java files.